src/openzaak/components/besluiten/models.py

```diff
@@ -324,3 +325,15 @@ def unique_representation(self):
             doc_identificatie = self.informatieobject.latest_version.identificatie
 
         return f"({besluit_repr}) - {doc_identificatie}"
+
+    def save(self, *args, **kwargs):
+
+        if (
+            self._informatieobject is not None
+            and self._informatieobject.latest_version is None
+        ):
+            raise ValidationError(
+                "Related InformatieObject must have at least one version"
+            )
```

Comment:
This does raise a 500 in case you save it via the admin, but tbh people shouldn't be doing this via the admin anyway, so I'll allow it :p